open-craft/cs_comments_service
# SE-3175 Fix codecov (ironwood)

See https://github.com/edx/cs_comments_service/pull/323

**Test instructions**:

- verify that the changeset matches https://github.com/edx/cs_comments_service/pull/323

**Reviewers**:
